### PR TITLE
DragDropHalogen - Simplify input in label

### PR DESCRIPTION
**Pull request by milesfrain**
Original:
```
label
  div (button style)
input
span
```

New:
```
label (button style)
  input
span
```

### CODE REVIEW

File: recipes/DragAndDropHalogenHooks/src/Main.purs [hunk 1]
```diff
@@ -69,40 +69,52 @@ hookComponent = Hooks.component \_ _ -> Hooks.do
           mbFileList = DataTransfer.files $ dataTransfer e
           fileArray = maybe [] FileList.items mbFileList
         Hooks.put filesIdx fileArray
+        Hooks.put hoverIdx false
```
Comment (milesfrain) [inline recipes/DragAndDropHalogenHooks/src/Main.purs:72]: Clears highlighting after file drop.

File: recipes/DragAndDropHalogenHooks/web/index.css [hunk 2]
```diff
@@ -1,4 +1,3 @@
 .otherCssNotInPurescript-Css {
-  background-color: lightblue;
   cursor: pointer;
```
Comment (milesfrain) [inline recipes/DragAndDropHalogenHooks/web/index.css:2]: We could get rid of this last CSS workaround once https://github.com/purescript-contrib/purescript-css/pull/94 is merged.
Then we'll be able to host this recipe on TPS.

File: recipes/DragAndDropHalogenHooks/src/Main.purs [hunk 3]
```diff
@@ -69,40 +71,53 @@ hookComponent = Hooks.component \_ _ -> Hooks.do
...
-    -- 2. add it to the DOM
-    -- 3. create a mouse event
-    -- 4. dispatch the mouse event to the input element
-    -- 5. (implication) file dialogue appears
-    -- 6. user selects a file
-    -- 7. input event handler runs a callback using user's selected file
-    -- 8. input element is removed from DOM
-    --
-    -- The approach used below is based on this SO answer:
-    -- https://stackoverflow.com/a/47094148
```
Comment (JordanMartinez) [inline recipes/DragAndDropHalogenHooks/src/Main.purs:84]: If you're going to delete this, maybe it should be moved to the recipe's Readme?

File: recipes/DragAndDropHalogenHooks/src/Main.purs (hunk 1 above)
Comment (JordanMartinez) [reply recipes/DragAndDropHalogenHooks/src/Main.purs:74]: Good point

File: recipes/DragAndDropHalogenHooks/web/index.css (hunk 2 above)
Comment (JordanMartinez) [reply recipes/DragAndDropHalogenHooks/web/index.css:2]: Sounds good!

File: recipes/DragAndDropHalogenHooks/src/Main.purs (hunk 3 above)
Comment (milesfrain) [reply recipes/DragAndDropHalogenHooks/src/Main.purs:84]: I just moved this section to the bottom of the file

Comment (JordanMartinez) [reply recipes/DragAndDropHalogenHooks/src/Main.purs:84]: Ah, I hadn't scrolled that far... Thanks for clarifyin!

**Pull request merged**